python/hail/tests/test_api.py

```diff
@@ -1186,6 +1186,19 @@ def test_make_table_empty_entry_field(self):
             t.row.dtype,
             hl.tstruct(**{'row_idx': hl.tint32, '0': hl.tint32, '1': hl.tint32}))
 
+    def test_transmute(self):
+        mt = (
+            hl.utils.range_matrix_table(1, 1)
+                .drop('row_idx', 'col_idx')
+                .annotate_globals(g1=0, g2=0)
+                .annotate_cols(c1=0, c2=0)
+                .annotate_rows(r1=0, r2=0)
+                .annotate_entries(e1=0, e2=0))
+        self.assertEqual(mt.transmute_globals(g3 = mt.g2 + 1).globals.dtype, hl.tstruct(g1=hl.tint, g3=hl.tint))
+        self.assertEqual(mt.transmute_rows(r3 = mt.r2 + 1).row.dtype, hl.tstruct(r1=hl.tint, r3=hl.tint))
+        self.assertEqual(mt.transmute_cols(c3 = mt.c2 + 1).col.dtype, hl.tstruct(c1=hl.tint, c3=hl.tint))
+        self.assertEqual(mt.transmute_entries(e3 = mt.e2 + 1).entry.dtype, hl.tstruct(e1=hl.tint, e3=hl.tint))
```

Comment:
Can you add force_counts to make sure this actually executes correctly?

Reply:
I can add force_count if you want, but I want to push back because the execution of this stuff depends on other machinery that's already tested (or should be tested separately). One of the reasons that our tests are slow is that we err on the side of making everything an integration test, instead of testing the functionality specific to the methods in the test name.

Reply:
ok.